client/src/document/toolbar/flaws.tsx

```diff
@@ -620,30 +620,28 @@ function Macros({
     };
   }, [opening]);
 
-  function openInEditor(msg: MacroErrorMessage, key: string) {
+  function openInEditor(msg: MacroErrorMessage, id: string) {
     const sp = new URLSearchParams();
     sp.set("filepath", msg.filepath);
     sp.set("line", `${msg.line}`);
     sp.set("column", `${msg.column}`);
     console.log(
       `Going to try to open ${msg.filepath}:${msg.line}:${msg.column} in your editor`
     );
-    setOpening(key);
+    setOpening(id);
     fetch(`/_open?${sp.toString()}`);
   }
 
   return (
     <div className="flaw flaw__macros">
       <h3>{humanizeFlawName("macros")}</h3>
-      {flaws.map((flaw) => {
+      {flaws.map((flaw, i) => {
```

Comment:
What's that `i` for? That's not used anywhere, is it?

Reply:
You're right! I thought I needed it and later realized that it wasn't necessary.